library/elixir

```diff
@@ -34,91 +50,91 @@ GitCommit: 0d9f47458468a8bf1407374731cbec077ab6f895
 Directory: 1.9/alpine
 
 Tags: 1.8.2, 1.8
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 4122b4840bd762d1434424e1ec693929b0198c98
 Directory: 1.8
 
 Tags: 1.8.2-slim, 1.8-slim
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 4122b4840bd762d1434424e1ec693929b0198c98
 Directory: 1.8/slim
 
 Tags: 1.8.2-alpine, 1.8-alpine
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 4122b4840bd762d1434424e1ec693929b0198c98
 Directory: 1.8/alpine
 
 Tags: 1.8.2-otp-22, 1.8-otp-22
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 6dc5ffd3b4c2915096887b45ba8e71d391ce2398
 Directory: 1.8/otp-22
 
 Tags: 1.8.2-otp-22-alpine, 1.8-otp-22-alpine
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 6dc5ffd3b4c2915096887b45ba8e71d391ce2398
 Directory: 1.8/otp-22-alpine
 
 Tags: 1.7.4, 1.7
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 2b7dd2845d27a6dad57bf0047b305375d6182402
 Directory: 1.7
 
 Tags: 1.7.4-slim, 1.7-slim
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 7c1f05ca3fd47bdc86cab3f0310068646a31dcac
 Directory: 1.7/slim
 
 Tags: 1.7.4-alpine, 1.7-alpine
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 2b7dd2845d27a6dad57bf0047b305375d6182402
 Directory: 1.7/alpine
 
 Tags: 1.6.6, 1.6
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 0936291249c7e11d4618a17a2b452045c9e6233a
 Directory: 1.6
 
 Tags: 1.6.6-slim, 1.6-slim
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 0936291249c7e11d4618a17a2b452045c9e6233a
 Directory: 1.6/slim
 
 Tags: 1.6.6-alpine, 1.6-alpine
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: 0936291249c7e11d4618a17a2b452045c9e6233a
 Directory: 1.6/alpine
 
 Tags: 1.6.6-otp-21, 1.6-otp-21
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
+Architectures: amd64, arm32v7, arm64v8, i386
 GitCommit: b57a72d04ddd1f1b4e2e3f5b70e44e37def4db31
 Directory: 1.6/otp-21
 
 Tags: 1.6.6-otp-21-alpine, 1.6-otp-21-alpine
-Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
-GitCommit: 084efbdda747411b3a41c231deaff03c437e5aad
+Architectures: amd64, arm32v7, arm64v8, i386
+GitCommit: 373b3a8017bd774b7d193818a326de0fde7f6733
 Directory: 1.6/otp-21-alpine
 
 Tags: 1.5.3, 1.5
```

Comment:
Any particular reason to keep these older versions listed here? It looks like 1.5 and 1.4 aren't even getting security patches any more (https://hexdocs.pm/elixir/compatibility-and-deprecations.html)  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)